src/main/java/ch/admin/bag/covidcertificate/sdk/core/decoder/chain/CertTypeService.kt

```diff
@@ -49,7 +49,7 @@ internal object CertTypeService {
 			}
 		}
 
-		return if (numContainedContent == 1) type else null
+		return if (numContainedContent >= 1) type else null
```

Comment:
The unit tests in `CertTypeServiceTest` need to be adjusted to this change

Reply:
Also, it might be worth documenting somewhere the order of importance that the individual entries in the certificate now hold. E.g. a certificate containing a test and a past infection entry is of type `RECOVERY` and will be shown as such in the UI, even if the test entry would be more recent